email/src/main/java/io/micronaut/email/Email.java

```diff
@@ -0,0 +1,433 @@
+            return this;
+        }
+
+        /**
+         *
+         * @param replyTo reply to contact
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder replyTo(@NonNull Contact replyTo) {
+            builder = builder == null ? EmailHeader.builder().replyTo(replyTo) : builder.replyTo(replyTo);
+            return this;
+        }
+
+        /**
+         *
+         * @param to Email recipient
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder to(@NonNull String to) {
+            builder = builder == null ? EmailHeader.builder().to(to) : builder.to(to);
+            return this;
+        }
+
+        /**
+         *
+         * @param to Email recipient
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder to(@NonNull Contact to) {
+            builder = builder == null ? EmailHeader.builder().to(to) : builder.to(to);
+            return this;
+        }
+
+        /**
+         *
+         * @param cc carbon copy recipient
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder cc(@NonNull Contact cc) {
+            builder = builder == null ? EmailHeader.builder().cc(cc) : builder.cc(cc);
+            return this;
+        }
+
+        /**
+         *
+         * @param bcc Blind carbon copy recipient
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder bcc(@NonNull Contact bcc) {
+            builder = builder == null ? EmailHeader.builder().bcc(bcc) : builder.bcc(bcc);
+            return this;
+        }
+
+        /**
+         *
+         * @param subject Email's subject
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder subject(@NonNull String subject) {
+            builder = builder == null ? EmailHeader.builder().subject(subject) : builder.subject(subject);
+            return this;
+        }
+
+        /**
+         *
+         * @param cc carbon copy email address
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder cc(@NonNull String cc) {
+            builder = builder == null ? EmailHeader.builder().cc(cc) : builder.cc(cc);
+            return this;
+        }
+
+        /**
+         *
+         * @param bcc blind carbon copy email address
+         * @return The Transactional Email Builder
+         */
+        @NonNull
+        public Builder bcc(@NonNull String bcc) {
+            builder = builder == null ? EmailHeader.builder().bcc(bcc) : builder.bcc(bcc);
+            return this;
+        }
+
+        /**
+         *
+         * @param trackLinks Whether email links should be tracked
+         * @return Email Header Builder
+         */
+        public Builder trackLinks(TrackLinks trackLinks) {
```

Comment:
sometimes it's nice to have shortucts for the particular enums e.g. also provide

```
builder.trackLinksInHtml()
builder.trackLinksInHtmlAndText()
builder.trackLinksInText()
```